### PR TITLE
Refactor for IX and Designer Interactions

### PR DESCRIPTION
**Pull request by nimbinatus**
Updating site architecture to use Markdown and Jekyll collections to make the pipeline easier for IX and for designers to contribute. Removed Semantic UI and the extra JS libraries from Semantic UI. Any extra libraries will be removed when the data visualization pages are switched to images instead of live HTML.

Fixes SURF-145 and SURF-489

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ A prototype of the Design System created Rackspace Experience Design. This site
 
 Ensure ruby is installed on your system and run the following.
 
-```
+``` bash
 bundle install
-jekyll serve --watch
+bundle exec jekyll serve --watch --config _config.yml,_dev-config.yml
```
Comment (nimbinatus) [inline README.md]: I think we could, for sure. Right now, this should work for this upcoming surge as the infodev folks can run with this, and I'll get it added to a JIRA task.

**Comment by technabors** [comment]
:shipit: 

TLDR: Couple of styling changes need to made post this PR to ensure everything still looks appropriate but we are going to merge and fix those later to get the markdown format in the hands of the Info Design folks.
------

I finally got a chance to look at the PR and found the following things that I wanted to discuss with you.
```- Figcaption spacing (tweak this slightly to align better)
- Section end Padding (need consistent padding here, sections are runnign together from end of one to start of another.)
- Note Styling (This is new, I would like to clean us the visuals a bit)
- Grid consistency (some offset things on nav page as an example)
- Status pill styling ( add an HTML class or change nesting in the SASS to fix)
- Status.html page (iterator no longer working, this is not really an issue because the page is hidden but needs to be fixed in time.)
- ToC Style (I didn't style these well when I built the janky JS but would like to update these now. Probably not a blocker)
```


[9:43] 
I can probably fix these things myself, but wanted to discuss with you to see which we need to fix prior to merge and which we will manage afterwards as part of the redesign.


Laura Santamaria
[9:44 AM] 
Kk; mainly I’m wanting to get the Markdown into the hands of the infodevs at the moment. The site redesign is happening this sprint (or, at least, barring ops issues over with infodev, it should be), so I’m not too concerned with getting the styling exactly right as the style will be iterated over a lot this sprint.


Evan Nabors [9:49 AM] 
Ok, so whats the timeline for you needing this pushed into prod for the ID team?


Laura Santamaria
[9:49 AM] 
The figure caption spacing and the section end padding will be part of the site redesign. The note styling was mainly a stop gap as I didn’t see any notes planned for in the site redesign; I don’t think they’ll be there in the next iteration as all of that should be held in JIRA. The grid is a bit off; I’m mainly waiting for the Helix grid to get settled so I can carry that to the site. The status pill stuff is actually going away in favor of a note on the family pages per Ty, so I’m not too inclined to change it just to remove it a week later. We certainly can, though, if it’s really a problem. The status page can get fixed with this next design so we can release it. And the TOC style can certainly be updated if you wanted; I figured we could handle it with the release.


[9:50] 
I was hoping to get it up to staging today; not sure about prod. I have a sync up with them in a bit.


[9:50] 
I should hear more about their timelines then


Evan Nabors
[9:53 AM] 
Ok how about this, there are only a couple of these issues that cause me a little heart burn [Section padding, Status Pills, Notes]. How about I give a LGTM :shipit: in GH and you push to staging. I’ll then branch staging and hammer out a couple stopgap CSS solutions to these tomorrow and file the work under https://jira.rax.io/browse/SURF-551 that way you don’t have to deal with it or feel like i’m slowing down progress for onboarding ID team.


[9:54] 
And I’ll paste this convo into the PR just for posterity’s sake

**Comment by technabors** [comment]
@nimbinatus One quick note. I don't think we have had the time to track down the issues with Travis autodeploy to staging (I believe this was SSH refusal or something), so this will require a manual publish if you want to see the results on the staging server. Let me know if you need my help with that.

**Comment by nimbinatus** [comment]
@technabors, I don't have access to the servers; I don't have the credentials. If you could do the manual publish (if we need to do it versus just pushing it up to production), that would be helpful, please. When Drew gets back, we can address that problem.
